Content.Server/Weapon/Melee/Esword/Components/EswordComponent.cs

```diff
@@ -0,0 +1,24 @@
+using Content.Shared.Sound;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+
+namespace Content.Server.Weapon.Melee.Esword
+{
+    [RegisterComponent]
+    internal class EswordComponent : Component
+    {
+        public override string Name => "Esword";
```

Comment:
The component name shouldn't be abbreviated. Just use `EnergySword` instead of `ESword`.

Having a component & entity-prototype with the same name doesn't cause issues..... or at least, it shouldn't

Reply:
Yeah